draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1454,7 +1452,7 @@ deployed and not likely to be registered in a timely manner otherwise.</t>
    was received as a token or a quoted string.
 </t>
 <t>
-   Historically, HTTP field values could be extended over multiple
+   Historically, HTTP/1.0 and HTTP/1.1 field values could be extended over multiple
    lines by preceding each extra line with at least one space or horizontal
    tab (<x:ref>obs-fold</x:ref>).
    <cref>This document assumes that any such <x:ref>obs-fold</x:ref> has been replaced with one or more
```

Comment:
we would have to fix this reference as well. I suggest using a ref to Messaging instead.

Reply:
I just removed "1.0 and HTTP/1.1" -- there's an appropriate reference to messaging in the next sentence.